<launch>
    <node pkg="narf_place_recognition" type="bag_to_dataset" name="bag_to_dataset" output="screen">
        <param name="bagFiles" type="string" value="/media/D/Datasets/PlaceRecognition/VelodyneForest/2015-05-28-11-51-12.bag /media/D/Datasets/PlaceRecognition/VelodyneForest/2015-05-28-13-38-32.bag"/>
        <param name="outputPath" type="string" value="/media/D/Datasets/PlaceRecognition/VelodyneForest/Output/"/>
        <param name="icpConfigPath" type="string" value="/home/smichaud/Workspace/NarfDadasetGeneration/narf_place_recognition/config/sick_icp.yaml"/>
        <param name="cloudTopic" type="string" value="/velodyne_points"/>
        <param name="poseTopic" type="string" value="/robot_pose_ekf/odom_combined"/>
        <param name="pointCloudKeepOneOutOf" type="int" value="80"/>
        <param name="isOdomMergedCloudsSaved" type="bool" value="true"/>
        <param name="isOdomOutput" type="bool" value="true"/>
    </node>
</launch>
